dymos/test/test_run_problem.py

```diff
@@ -831,12 +831,6 @@ def test_simulate_array_param(self):
 
         assert_near_equal(sol - sim, np.zeros_like(sol))
 
-        # Test that the parameter is available in the solution and simulation files
```

Comment:
This was removed because we don't record inputs anymore?

Reply:
Yep